library/src/main/java/com/novoda/downloadmanager/LiteDownloadMigrationService.java

```diff
@@ -62,12 +77,24 @@ public void onCreate() {
             public void onUpdate(String message) {
                 updateMessage = message;
                 if (migrationCallback != null) {
+                    MigrationStatus migrationStatus = new VersionOneToVersionTwoMigrationStatus(message);
+                    updateNotification(migrationStatus);
                     migrationCallback.onUpdate(message);
                 }
             }
         };
     }
 
+    private void updateNotification(MigrationStatus migrationStatus) {
+        NotificationInformation notification = notificationCreator.createNotification(channelCreator.getNotificationChannelName(), migrationStatus);
+        startForeground(notification.getId(), notification.getNotification());
+
+        if (migrationStatus.message().equals("Migration Complete")) {
```

Comment:
Can we transform the strings on this migration status object by an enum `Status`?

Reply:
I've made an enum `Status` on the follow-up branch to this one. I just wanted to start introducing the foreground service in this PR, following it up with improvements to the status and notifications 😄